src/common/CommonUtils.ts

```diff
@@ -19,6 +20,24 @@ export class CommonUtils {
         return Promise.resolve();
     }
 
+    public static delay(ms: number): Promise<void> {
+        return new Promise((resolve) => setTimeout(resolve, ms));
+    }
+
+    public static resolvePath(inputPath: string): string {
```

Comment:
Can we rename this to `resolveUserHomePath`?

Reply:
well it's not just resolving the home path, that's just one of the things it does.
1. Checks for and resolves the home path
2. Then checks to resolve the path (i.e. if it is relative then it is resolved to an absolute path)
3. Then does a normalization of the final path before returning it.

I couldn't come up with a better name for this function though 😄